hookman/hookman_generator.py

```diff
@@ -284,6 +285,9 @@ def generate_plugin_package(
 
         :param Dict[str,str] extras_defaults:
             (key, value) entries to be added to "extras" if not defined by the original input yaml.
+
+        :param Optional[str] package_name_extra:
```

Comment:
```suggestion
        :param Optional[str] package_name_suffix:
```

Reply:
I was originally planning to use `package_name_suffix` =P

- [ ] name `package_name_extra` as `package_name_suffix`;